frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -79,7 +59,7 @@ export const SearchForm = ({
         }}
         data-testid="search-form-input-area"
       >
-        Search for an area by location or organisation
+        Search for an area
```

Comment:
Shouldn't this say?:
```suggestion
        Search by area
```